plugin-server/tests/worker/vm.test.ts

```diff
@@ -1021,6 +1021,37 @@ describe('vm tests', () => {
         })
     })
 
+    test('posthog.capture accepts groups', async () => {
```

Comment:
Note: We've been trying to move away from integration-only test model. They are hard to debug and often bugs slip through.

With this in mind, I'd leave this test be, but could you add also a unit test file for extensions/posthog.ts with your new test?